pyproject.toml

```diff
@@ -12,7 +12,7 @@ python-dotenv = "^0.13.0"
 # Raspberry Pi uses the ARM platform, so these will only install on a Pi
 picamera = { version = "^1.13", markers = "platform_machine == 'armv7l'" }
 "RPi.GPIO" = { version = "^0.7.0", markers = "platform_machine == 'armv7l'" }
-mhp = {git = "https://github.com/monash-human-power/common.git", rev="202008.4"}
+mhp = {git = "https://github.com/monash-human-power/common.git", rev="202008.5"}
```

Comment:
It seems like we've been inconsistent with our tag names. In order to run `poetry update` right now, this revision has to be `v202008.0005`. We should decide on which format we prefer - I prefer the shorter one, less to type and also consistent with the package version.